draft-ietf-ppm-dap.md

```diff
@@ -1440,21 +1470,17 @@ struct {
 } InputShareAad;
 ~~~
 
-The Leader responds to well-formed requests with HTTP status code 201
-Created. Malformed requests are handled as described in {{errors}}.
-Clients SHOULD NOT upload the same measurement value in more than one report if
-the Leader responds with HTTP status code 201 Created.
+If the upload request is malformed, the Leader aborts with error
+`invalidMessage`.
 
-If the Leader does not recognize the task ID, then it MUST abort with error
-`unrecognizedTask`. Note that indicating an unrecognized task might indicate
-that the Client is misconfigured or the task has expired.
```

Comment:
I don't think the note is helpful, in the sense that it doesn't tell implementers what their implementation should do.